tests/drivers/build_all/sensor/prj.conf

```diff
@@ -105,3 +105,4 @@ CONFIG_INA230=y
 CONFIG_INA237=y
 CONFIG_MHZ19B=y
 CONFIG_MAX31875=y
+CONFIG_AM2301B=y
```

Comment:
This isn't necessary anymore since the default now corresponds to whether there is a compatible devicetree node enabled.

Reply:
fixed it